test/functional/sqlsrv/MsCommon.inc

```diff
@@ -110,6 +98,8 @@ function startTest($testName)
  */
 function endTest($testName)
 {
+    // reset the utf-8 flag
+    setUTF8Data(false);
```

Comment:
I don't think we should do this here as this is a generic function it doesn't necessary have anything to do with UTF8 testing. The tests which care about this should set this themselves

Reply:
ok